examples/channels-correlation/channels_correlation_service.out

```diff
@@ -0,0 +1,5 @@
+# To start the service, navigate to the directory that contains the
+# `.bal` file and use the `ballerina run` command.
```

Comment:
minor correction:
"# To start the service, navigate to the directory that contains the
# `.bal` file, and then use the `ballerina run` command."